lib/pymedphys/tests/dicom/test_header_tweaks.py

```diff
@@ -41,6 +42,11 @@
 )
 
 
+def get_pymedphys_dicom_cli():
```

Comment:
DRY

Reply:
What do you mean here sorry?

Reply:
this function is being placed in multiple separate files.
So I'm asking that that you not repeat the definition, and instead put it in one place, import from that, and invoke it.

Reply:
One is `get_pymedphys_dicom_cli`, the other `get_pymedphys_dicom_anonymise_cli`, the purpose of both is more of a file specific macro to include `python -m pymedphys` at the beginning of the call.

Reply:
Nevermind :), it is in two places :) I can pull it out.